src/main/java/SearchFile.java

```diff
@@ -0,0 +1,44 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+public class SearchFile {
+
+	public static void main(String[] args) {
+		try {
+
+			Scanner input = new Scanner(System.in);
+			String[] command = input.nextLine().split(" ");
+
+			if (!command[0].equals("search"))
+				return;
+
+			String keyword = command[1];
```

Comment:
Not dynamic due to hard coded arguments. What will you do if you want to add more arguments?